src/graph/util/ExpressionUtils.cpp

```diff
@@ -247,7 +247,10 @@ Expression *ExpressionUtils::rewriteInExpr(const Expression *expr) {
   DCHECK(expr->kind() == Expression::Kind::kRelIn);
   auto pool = expr->getObjPool();
   auto inExpr = static_cast<RelationalExpression *>(expr->clone());
-  auto containerOperands = getContainerExprOperands(inExpr->right());
+  std::vector<Expression *> containerOperands;
+  if (inExpr->kind() == Expression::Kind::kRelIn) {
+    containerOperands = getContainerExprOperands(inExpr->right());
```

Comment:
if `containerOperands` is empty, the `orExpr` might not construct correctly. Besides, `rewriteInExpr` require a `IN` expression.